test/integration/scheduler/scheduler_test.go

```diff
@@ -289,7 +287,6 @@ priorities: []
 					},
 				},
 			}),
-			scheduler.WithBindTimeoutSeconds(defaultBindTimeout),
```

Comment:
In theory, we should be replacing this with `scheduler.WithProfiles` where you would pass a PluginConfig. In reality, these are integration tests. Do we have integration tests around volume binding that we have faked to run fast?

Reply:
it's strange to configure the timeout here because it verifies policy config map only

anyway, I've updated the PR to use `scheduler.WithProfiles` here not to change the logic

> Do we have integration tests around volume binding that we have faked to run fast?
 
pods not using volumes are ignored by VolumeBinding plugin and always run fast
 
volume-related integration tests reside in `test/integration/volumescheduling` pkg, but we didn't use customized volume binding timeout. maybe we can add some negative tests in the future